.github/CONTRIBUTING.md

```diff
@@ -62,6 +64,20 @@ If you want to create a kata for some topic, start by checking the roadmap to se
 If somebody is already working on this topic, you can try to find them (using the repository issues) and coordinate with them.
 If the topic you want is not claimed, or is not on the list, go ahead and let us know you'll be working on it by creating an issue.
 
+### Testing the Katas
+
+You can use Jupyter notebook front-end of the kata you're working on to validate the kata (i.e., to check that all tasks have correct reference solutions for them, and that all tests used in the notebook actually exist in the project.
```

Comment:
```suggestion
You can use Jupyter Notebook front-end of the kata you're working on to validate the kata (i.e., to check that all tasks have correct reference solutions for them, and that all tests used in the notebook actually exist in the project.
```
I believe your capitalization in the bit with "...in the notebook actually exist in the project" is correct, though, in that "Jupyter Notebook" is the name for a specific Jupyter client and for the data format used by that client and is hence a proper noun, while "notebook" is not a proper noun.

Reply:
I suspected I was not consistent with my capitalization somewhere... Thank you!